### ISSUE TITLE
DetailLabel and TitleLabel layout is NOT set well

### ISSUE THREAD

**Issue by PCBZ** [dzenbot/DZNEmptyDataSet#479]
For the setting constraints of detailLabel or titleLabel, 
` if ([self canShowDetail]) {
            
            [subviewStrings addObject:@"detailLabel"];
            views[[subviewStrings lastObject]] = _detailLabel;
            
            [self.contentView addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"H:|-(padding@750)-[detailLabel(>=0)]-(padding@750)-|"
                                                                                     options:0 metrics:metrics views:views]];
        }`
the leading and trailing constraints priority are not .default as 1000, which make our label will be exceed the superView.
Could we set it to 1000 from 750 ?

**Comment by everlof** [comment]
I agree with this. I've had the same issue and this is also how I solved it.
